### PR TITLE
MOSIP-35957 uniqueIdentifier for masterData

### PR DESCRIPTION
**Pull request by maheswaras**
Added unique identifier for first 113 API testcases

### CODE REVIEW

File: api-test/src/main/resources/masterdata/Device/CreateDevice.yml [hunk 1]
```diff
@@ -27,12 +28,13 @@ CreateDevice:
 	"isActive": false,
 	"macAddress": "D4-3D-7E-C0-8C-4A",
 	"zoneCode": "$IGNORE$",
-	"validityDateTime": "$IGNORE$",
+	"validityDateTime": "$TIMESTAMP$",
```
Comment (mohanachandran-s) [inline api-test/src/main/resources/masterdata/Device/CreateDevice.yml:31]: This should be unchanged

File: api-test/src/main/resources/masterdata/Device/CreateDevice.yml [hunk 2]
```diff
@@ -59,12 +61,13 @@ CreateDevice:
 	"isActive": false,
 	"macAddress": "D4-3D-7E-C0-8C-4A",
 	"zoneCode": "$IGNORE$",
-	"validityDateTime": "$IGNORE$",
```
Comment (mohanachandran-s) [inline api-test/src/main/resources/masterdata/Device/CreateDevice.yml:62]: This should be unchanged

File: api-test/src/main/resources/masterdata/Device/UpdateDevice.yml [hunk 3]
```diff
@@ -59,12 +61,13 @@ UpdateDevice:
 	"isActive": "$IGNORE$",
 	"macAddress": "D4-3D-7E-C0-8C-4A",
 	"zoneCode": "$IGNORE$",
-	"validityDateTime": "$IGNORE$",
```
Comment (mohanachandran-s) [inline api-test/src/main/resources/masterdata/Device/UpdateDevice.yml:62]: This should be unchanged

File: api-test/src/main/resources/masterdata/Device/UpdateDevice.yml [hunk 4]
```diff
@@ -27,12 +28,13 @@ UpdateDevice:
...
```
Comment (mohanachandran-s) [inline api-test/src/main/resources/masterdata/Device/UpdateDevice.yml:30]: This should be unchanged

**Review by mohanachandran-s — CHANGES_REQUESTED** [review]

File: api-test/src/main/resources/masterdata/Device/CreateDevice.yml (hunk 1 above)
Comment (maheswaras) [reply api-test/src/main/resources/masterdata/Device/CreateDevice.yml]: Updated properly

**Review by mohanachandran-s — APPROVED** [review]

**Review by ase-101 — APPROVED** [review]

**Pull request merged**